app/routes/customer_routes.py

```diff
@@ -0,0 +1,134 @@
+from flask import Blueprint, jsonify, make_response, request, abort
+from werkzeug.exceptions import RequestEntityTooLarge
+from app import db
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rental import Rental
+from datetime import datetime, timezone, timedelta
+
+customer_bp = Blueprint("customer", __name__, url_prefix="/customers")
+
+# HELPER FUNCTIONS
+def valid_int(number, parameter_type):
+    try:
+        number = int(number)
+    except:
+        abort(400, {"error":f"{parameter_type} must be an int"})
+
+def get_customer_from_id(customer_id):
+    valid_int(customer_id, "customer_id")
+    customer = Customer.query.get(customer_id)
+    if not customer:
+        abort(make_response({"message":f"Customer {customer_id} was not found"}, 404))
+    
+    return customer
+
+# CUSTOMER ROUTES
+
+# Get all customers
+@customer_bp.route("", methods =["GET"])
+def read_all_customers():
+    sort_query = request.args.get("sort")
+
+    # sort by name
+    if sort_query == "name":
+        customers = Customer.query.order_by(Customer.name.asc())
+    # sort asc by registered_at, postal_code
+    elif sort_query == "registered_at":
+        customers = Customer.query.order_by(Customer.registered_at.asc())
+    elif sort_query =="postal_code":
+        customers = Customer.query.order_by(Customer.postal_code.asc())
+    else:
+        customers = Customer.query.all()
+
+    customers_response = []
+    for customer in customers:
+        customers_response.append(
+            customer.to_dict()
+        )
```

Comment:
This could be a nice place to use a list comprehension

```suggestion
    customers_response = [customer.to_dict() for customer in customers]
```